tests/client/test_async_client.py

```diff
@@ -18,7 +19,9 @@ async def test_get(server):
     assert response.http_version == "HTTP/1.1"
     assert response.headers
     assert repr(response) == "<Response [200 OK]>"
-    assert response.elapsed > timedelta(seconds=0)
+    if sys.platform != "win32":
+        # flaky on windows
+        assert response.elapsed > timedelta(seconds=0)
```

Comment:
Thanks for highlighting flaky areas on our tests, windows.

I think this is a bit of code smell either way. Perhaps we switch it to this?...

```suggestion
    assert response.elapsed >= timedelta(seconds=0)
```

Reply:
isn't this a no-op?

Reply:
It's an odd and flaky thing to test against.
I think it only ended up in there because of code coverage.
We could comment it with something like this perhaps?

```python
# Sanity test the property access and return type.
```